casepro/cases/models.py

```diff
@@ -258,37 +258,62 @@ def search(cls, org, user, search):
         return queryset.order_by('-opened_on')
 
     @classmethod
-    def get_or_open(cls, org, user, message, summary, assignee, user_assignee=None):
+    def get_or_open(cls, org, user, message, summary, assignee, user_assignee=None, contact=None):
+        """
+        Get an existing case, or open a new case if one doesn't exist. If message=None, then contact is required, and
+        any open case for that contact will be returned. If no open cases exist for the contact, a new case will be
+        created.
+        """
```

Comment:
Resolved in https://github.com/praekelt/casepro/pull/109/commits/ef6a4a40cd8161ccd4366ad74ccc0744431502d9